Various UI improvements and a bug fix.

ronickg opened the pull request:
Bug fixes:

- fix the amount and fee values not correctly updating in send page when pressing onSubmit

UI improvements:

- Show addresses in this format now  "txch1fz...agzfzg" which is responsive to the width
- Added masked inputs for fee or amount inputs which prevent inputing a smaller decimal then decimals or default of 12.
- Added a warning to confirmation dialog if the users fee is very high

Rigidity commented on src-tauri/gen/android/app/build.gradle.kts:
```diff
@@ -25,20 +25,20 @@ android {
-            if (keystorePropertiesFile.exists()) {
-                keystoreProperties.load(FileInputStream(keystorePropertiesFile))
-            }
+    // signingConfigs {
```
Why is this commented out?

Rigidity commented on src-tauri/gen/android/app/src/main/res/values/themes.xml:
```diff
@@ -1,6 +1,10 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
     <style name="Theme.sage_tauri" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+      <!-- <item name="android:statusBarColor">@android:color/transparent</item> -->
```
What is this for?

Rigidity commented on src-tauri/gen/apple/sage-tauri.xcodeproj/project.pbxproj:
```diff
@@ -355,7 +355,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = NQJQRYZZG3;
+				DEVELOPMENT_TEAM = 5PSQ3NC8JP;
```
Does this need to be changed?

Rigidity commented on src/components/ConfirmationDialog.tsx:
```diff
@@ -127,6 +127,16 @@ export default function ConfirmationDialog({
                   </TabsTrigger>
                 </TabsList>
                 <TabsContent value='simple'>
+                  {/* {isHighFee && !fee.isZero() && ( */}
```
What is this comment for?

Rigidity commented on src/components/Layout.tsx:
```diff
@@ -43,8 +45,9 @@ export default function Layout(props: PropsWithChildren<object>) {
-              <img src={icon} className='h-6 w-6' />
-              {wallet?.name}
+              <img src={icon} className='h-8 w-8' />
+              {/* {wallet?.name} */}
```
Do we need this still

ronickg commented:
Went over the comments, removed and reverted not needed changes. Fixed the fee is to high warning to use mojo's now as well after the merge. Also fixed the copybox to have a value and dispValue, otherwise the copy will use the short format address.
